src/hist.jl

```diff
@@ -259,34 +272,48 @@ end
 append!{T,N}(h::AbstractHistogram{T,N}, vs::NTuple{N,AbstractVector}, wv::WeightVec) = append!(h, vs, values(wv))
 
 
+# Turn kwargs nbins into a type-stable tuple of integers:
+function _nbins_tuple{N}(vs::NTuple{N,AbstractVector}, nbins)
+    template = map(length, vs)
+
+    if VERSION < v"0.6.0-"
```

Comment:
Use a version on which you know this works, for example the one you're using. What we should avoid is a too broad check which would trigger failures on old 0.6 versions.